packages/babel-plugin-transform-es2015-template-literals/src/index.js

```diff
@@ -18,7 +18,9 @@ export default function ({ types: t }) {
         let raw = [];
 
         for (const elem of (quasi.quasis: Array)) {
-          strings.push(t.stringLiteral(elem.value.cooked));
+          if (elem.value.cooked !== null) {
```

Comment:
@hzoo `var undefined = 0` works at least in sloppy mode, so you do need `void 0 or equivalent. Otherwise lgtm.